client/src/containers/PostDashboard.tsx

```diff
@@ -78,9 +100,21 @@ export default function PostDashboard() {
   const [pageCount, setPageCount] = React.useState(1);
   const [page, setPage] = React.useState(1);
 
+  const useQuery = (): [string, (q: string) => void] => {
+    const [query, setQuery] = React.useState('');
+
+    const setEscapedQuery = (q: string) => {
+      setQuery(q.replace(/\W+/g, '&'));
```

Comment:
Is this the only place we sanatize user input? Shouldn't we also do this in the backend, since nothing is stopping the user from directly calling our route